docs/tutorials/avro.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p><code>import json</code> is a duplicate import.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
thanks, updated.